restaurants/src/app/app-routing.module.ts

```diff
@@ -1,7 +1,13 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { RestaurantDetailsComponent } from './components/restaurantdetails/restaurantdetails.component';
+import { RestaurantsComponent } from './components/restaurants/restaurants.component';
 
-const routes: Routes = [];
+const routes: Routes = [
+  {path: '', redirectTo: '/dashboard', pathMatch: 'full' },
```

Comment:
Don't think `/dashboard` is a valid uri path that exists right now - instead, thinking you'd want the root path of `''` to redirect to your restaurants page instead? At some point you'll probably create a nice home page, but until then, seems like the /restaurants page is acting as your home page.

Reply:
done